index.html

```diff
@@ -114,6 +114,38 @@ <h2>A project for first-time contributions</h2>
           <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
           <!-- Keep one line of space above and below your card -->
           <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+
+          <!-- ________ ooooshino card START ________  -->
+          <div class="card">
+            <p class="name">ooooshino</p>
+            <p class="contact">
+                <i class="fab fa-github"></i>
+                <a href="https://github.com/ooooshino" target="_blank">ooooshino</a>
+            </p>
+            <p class="about">
+
+            <ul>
+                <li>🎯 A curious and passionate pre-final year Software Engineering  student of Neusoft </li>
+                <li>👨‍💻 Front-End developer & love to do some nice and cool UI/UX</li>
+                <li>🚀 Currently learning Three.js</li>
+            </ul>
+            </p>
```

Comment:
```suggestion
            <p class="about">A curious and passionate pre-final year Software Engineering  student of Neusoft. Front-End developer & love to do some nice and cool UI/UX. Currently learning Three.js</p>
```

Avoid using html tags other than given on the template. It may break while we archive your card.